content/02.introduction.md

```diff
@@ -24,69 +24,58 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
-
-### SARS-CoV-2: Origin, Phylogenetics, and Evolution
-
-#### Origin of SARS-CoV-2
-The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
-Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
-
-Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
-This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
-
-### COVID-19: Mechanisms and Presentation
-
-#### Immunological Mechanisms of Coronavirus-driven Disease in Humans
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19. 
+The human immune response is double-pronged, with the innate and adaptive immune responses playing distinct roles.
+Through innate immunity, the body can rapidly recognize and attack pathogens and infected cells. 
+Cells from innate immunity also pass critical information to prime the adaptive immune response.
+Adaptive immunity can be broadly classified into the humoral and cell-mediated responses. 
+These are distinct from the innate immune responses because they target specific protein sequences from pathogens. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells, and can also mediate killing of pathogen-infected cells. 
+Cell-mediated responses include the killing of pathogen-infected cells by cytotoxic CD8+ T cells, and the secretion of cytokines to drive inflammation and support immune cell activities.
+The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
```

Comment:
This/after line 41 might be a  place where we could mention long-term immunity. Maybe a sentence like:
Both of these responses have mechanisms for long-term immunity through memory B-cells or memory T-cells.
OR, you could put such a sentence after line 38 "These are distinct... specific protein sequences from pathogens." with maybe a sentence like:
Adaptive immunity also has mechanisms for long-term immunity through memory B-cells or memory T-cells.